community-operators/hazelcast-enterprise/hazelcast-operator.v0.0.1.clusterserviceversion.yaml

```diff
@@ -0,0 +1,207 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  annotations:
```

Comment:
Please add `metadata.annotations.repository` to both CSVs. This should be a link to the operator's source code repository

Reply:
Added.